src/frontend.md

```diff
@@ -10,6 +10,14 @@
 
 ## Project structure
 
+Project structure should follow next principles:
+
+1. Not needed to move files too often.
```

Comment:
Could you please explain in what cases the project structure could help avoid moving files?

Reply:
Let's imagine that we store files hierarchically by their usage and have such a component:
`src/pages/account/AccountCreation/components/AccountCategorySelect`
and then wanna add new page `src/pages/account/AccountEdition` and use there `AccountCategorySelect`.
So in this case we should move it to another place (in example `src/components/account/AccountCategorySelect`) since it's used in multiple places.

If we group files by their own properties, but not by their relation to another files, we wouldn't move this file when we reuse it, because it's placed in `src/components/account/AccountCategorySelect` regardless of it's usage. We would move it only in case of changing purpose of this file, for example changing domain: `AccountCategorySelect` -> `BankCategorySelect` (don't try to understand purpose of this component, it's just for example), or changing control type `AccountCategorySelect` -> `AccountCategoryInput`.

Reply:
> Not needed to move files too often

Maybe we should make a formulating more strict by remove "too often".

Reply:
I tried to analyze existing MFs, and the main issue with that we almost don't move them now :D